### DISCUSSION TITLE
get_potential_energies() from calculator

### DISCUSSION THREAD

**Discussion by carajillu** [ACEsuit/mace#759] [Q&A] [answered]
A silly one, probably, but I've seen that the MACE calculator has a placeholder for a get_potential_energies() function. Is that meant to return a vector with the potential energy of each atom in the system? If so, is there any timeline for it to be implemented?

**Comment by ilyes319** [comment, marked as answer]
Hello @carajillu, if you want to access that, you can use the `node_energies` that outputs exactly that (in the result dict).
